docs/api-reference/endpoints/pki/syncs/options.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: "List PKI Sync Options"
+openapi: "GET /api/v1/pki/syncs/options"
+---
+
+<Info>
+This endpoint lists all available PKI sync destination options and their capabilities.
+</Info>
+
+## Request
+
+<ParamField query="projectId" type="string" optional>
+  Project ID (for authorization purposes, but the options are global)
+</ParamField>
```

Comment:
what do you mean by "options are global" here? 

Maybe there's no need to check for the PKI sync permissions to get the sync options? I think we do it like for the secret sync - not sure